package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "dbt-styleguide",
+  "version": "0.0.1",
+  "description": "Styleguide for dbt",
+  "main": "ui/js/scripts.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "release": "release-it"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/fishtown-analytics/dbt-styleguide.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/fishtown-analytics/dbt-styleguide/issues"
+  },
+  "homepage": "https://github.com/fishtown-analytics/dbt-styleguide#readme",
+  "devDependencies": {
+    "auto-changelog": "^2.2.0",
+    "release-it": "^13.6.6"
+  },
+  "release-it": {
+    "github": {
+      "release": false
```

Comment:
For automating a github release, the config needs a [personal access token](https://github.com/release-it/release-it/blob/master/docs/github-releases.md). If we want to, we could grant developers these tokens and automate this process as well. Thoughts?